docker-compose.yml

```diff
@@ -152,11 +154,33 @@ services:
     restart: unless-stopped
     ports:
       - 7777:7777
+    environment:
+      RABBITMQ_PORT_5672_TCP_ADDR: 192.168.168.167
+    env_file:
+      - .docker-compose.wb.env
+    volumes:
```

Comment:
I've got the celery wb_worker depending on rabbitmq, does the the wb service need to be dependent on it as well?